src/index.tsx

```diff
@@ -65,34 +72,76 @@ const RenderViewer: React.FC<Props> = ({
   }, [activeCanvas]);
 
   useEffect(() => {
-    // Update with user config options
+    if (activeManifest)
+      vault
+        .loadManifest(activeManifest)
+        .then((data: any) => {
+          setManifest(data);
+          dispatch({
+            type: "updateActiveCanvas",
+            canvasId: data.items[0] && data.items[0].id,
+          });
+        })
+        .catch((error: any) => {
+          console.error(`Manifest failed to load: ${error}`);
+        })
+        .finally(() => {
+          dispatch({
+            type: "updateIsLoaded",
+            isLoaded: true,
+          });
+        });
+  }, [activeManifest]);
+
+  useEffect(() => {
+    let resource = id;
```

Comment:
Could you do `const resource = id ? id : manifestId`, or something like that maybe?